packages/video-intelligence/src/index.js

```diff
@@ -14,41 +14,59 @@
-  v1beta1: require('./v1beta1/index.js'),
-};
+var v1beta1 = require('./v1beta1/index.js');
 
 const VERSION = require('../package.json').version;
 
-
 /**
- * Create an imageAnnotatorClientfor the appropriate version.
- *
- * @param {Object=} opts - The optional parameters.
- * @param {String=} opts.servicePath
- *   The domain name of the API remote host.
- * @param {number=} opts.port
- *   The port on which to connect to the remote host.
- * @param {grpc.ClientCredentials=} opts.sslCreds
- *   A ClientCredentials for use with an SSL-enabled channel.
- * @param {Object=} opts.clientConfig
- *   The customized config to build the call settings. See
- *   {@link gax.constructSettings} for the format.
+ * [Google Cloud Video Intelligence API](https://cloud.google.com/video-intelligence)
+ * makes videos searchable, and discoverable, by extracting metadata with an
+ * easy to use REST API. You can now search every moment of every video file in
+ * your catalog and find every occurrence as well as its significance. It
+ * quickly annotates videos stored in Google Cloud Storage, and helps you
+ * identify key nouns entities of your video, and when they occur within the
+ * video. Separate signal from noise, by retrieving relevant information at the
+ * video, shot or per frame.
+ *
+ * <h2>This is an auto-generated API</h2>
+ *
+ * It does not follow the conventions you're familiar with from other parts of
+ * our library. A handwritten layer is not yet available.
+ *
+ * The example below shows you how to instantiate the generated client. For
+ * further documentation, please browse the
+ * [Video Intelligence .proto files](https://github.com/googleapis/googleapis/blob/master/google/cloud/videointelligence/v1beta1/video_intelligence.proto)
+ * on GitHub.
+ *
+ * @constructor
+ * @alias module:video-intelligence
+ *
+ * @resource [Cloud Video Intelligence](https://cloud.google.com/video-intelligence)
+ *
+ * @param {object} options - [Configuration object](#/docs).
+ * @param {number=} options.port - The port on which to connect to the remote
+ *     host.
+ * @param {string=} options.servicePath - The domain name of the API remote
+ *     host.
  */
-function videointelligenceV1beta1(opts) {
+function VideoIntelligence(options) {
   // Define the header options.
-  opts = opts || {};
-  opts.libName = 'gccl';
-  opts.libVersion = VERSION;
+  options = options || {};
+  options.libName = 'gccl';
+  options.libVersion = VERSION;
 
   // Create the image annotator client with the provided options.
-  var client = gapic.v1beta1(opts).videoIntelligenceServiceClient(opts);
+  var client = v1beta1(options).videoIntelligenceServiceClient(options);
   return client;
 }
 
 // The default export should be the latest version.
 // Assign all versions as version properties on the default.
-module.exports = videointelligenceV1beta1;
-module.exports.v1beta1 = videointelligenceV1beta1;
+module.exports = VideoIntelligence;
+module.exports.v1beta1 = v1beta1;
```

Comment:
This is how we usually do it-- we export the raw gapic module as the version number, without a wrapper.